flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -74,6 +78,24 @@ public Environment() {
 		this.deployment = DeploymentEntry.DEFAULT_INSTANCE;
 	}
 
+	public Map<String, ModuleEntry> getModules() {
+		return modules;
+	}
+
+	public void setModules(List<Map<String, Object>> modules) {
```

Comment:
Can we rename the input arg so as to make the code cleaner?

Reply:
Did you address this comment?

Reply:
it conforms to naming in other APIs of the class, so we should be good